test/.vimrc.example

```diff
@@ -12,5 +12,5 @@
 
 augroup black_on_save
   autocmd!
-  autocmd FileType python autocmd BufWritePre * Black
+  autocmd BufWritePre * if &filetype == 'python'|execute ':Black'|endif
 augroup end
```

Comment:
Did not notice this earlier, but should the file actually be under `hack/` instead of `test/` ?